packages/Header/Header.vue

```diff
@@ -19,7 +28,7 @@ export default {
   name: "Header",
   props: {
     title: { type: String, required: true },
-    description: { type: String },
+    description: { type: String, required: true },
```

Comment:
why is it required? we could have a header with no description no?

Reply:
We could yes, for the ATD for example. I just realized that so i will update this to remove the required.